lib/utils.pm

```diff
@@ -421,7 +421,7 @@ sub workaround_type_encrypted_passphrase {
     # If the encrypted disk is "just activated" it does not mean that the
     # installer would propose an encrypted installation again
     return if get_var('ENCRYPT_ACTIVATE_EXISTING') && !get_var('ENCRYPT_FORCE_RECOMPUTE');
-    record_soft_failure 'workaround https://fate.suse.com/320901' if sle_version_at_least('12-SP4');
+    record_soft_failure 'workaround https://fate.suse.com/320901' if is_sle('12-SP4+');
```

Comment:
that one is used on openSUSE for sure but as the fate is about SLE and we can not work with "feature commitment" for openSUSE I am fine how it is.

Reply:
There is an option to use `!is_sle('<12-sp4')`, but I would avoid it.
Using `|| is_opensuse` - is much more obvious.